src/masoniteorm/query/grammars/MySQLGrammar.py

```diff
@@ -219,3 +219,6 @@ def where_null_string(self):
 
     def where_not_null_string(self):
         return " {keyword} {column} IS NOT NULL"
+
+    def truncate_table_string(self):
+        return "TRUNCATE TABLE {table}"
```

Comment:
I just realized we need to support the whole enable / disable foreign keys thing. Tables won't be able to be truncated if they have foreign key constraints to other tables.  Typically when you want to truncate a table you want to disable this check.

See the platform classes as an example. We can honestly just copy and paste the method from the schema platform classes

Reply:
See here:

```python
return [
            self.disable_foreign_key_constraints(),
            f"TRUNCATE {self.wrap_table(table)}",
            f"DELETE FROM {self.wrap_table(table)}",
            self.enable_foreign_key_constraints(),
        ]
```

Reply:
I added foreign keys handling for SQLite only (just to check), is this what you meant ?

I found weird that passing foreign_keys=True will disable foreign_keys handling I would have done it the other way around but that's how it's done in Schema. 🤔

Reply:
@josephmancuso I posted this above. I updated only SQLite so that is why other tests are failing. Can you confirm what I have done is okay for SQLite ?

Reply:
Oh sorry I didn't see that. Yeah what you did in sqlite is correct